### PR TITLE
BI-30:khanhvnv - Doing exercise for printing out an interesting square with O, + and - 

### PR DESCRIPTION
**Pull request by khanhvnv**
## Exercise

* [Exercise-30](https://github.com/NT-Line-team/Behave_test/issues/30)

## Overview
This exercise is for practicing for loop and if statements.

Write the program that allow user to input N, then print the square as below:
Please input N:10
```
O  O  O  O  O  O  O  O  O  O  
O  O  -  -  -  -  -  -  -  O 

O  +  O  -  -  -  -  -  -  O 

O  +  +  O  -  -  -  -  -  O 

O  +  +  +  O  -  -  -  -  O 

O  +  +  +  +  O  -  -  -  O 

O  +  +  +  +  +  O  -  -  O 

O  +  +  +  +  +  +  O  -  O 

O  +  +  +  +  +  +  +  O  O 

O  O  O  O  O  O  O  O  O  O 
```
### Docs

### CODE REVIEW

**Comment by khanhvnv** [comment]
Hi Hung,

Please help to review this code for exercise 30.

Many thanks,
Khanh

File: python_basic/khanhvnv/BI_20_KhanhVu_managing user accounts simple system.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	fsuccess = 0
+	name = raw_input('Login - Enter username: ')
+	passwrd = raw_input('Login - Enter password: ')
+	
+if fsuccess == 1:
+	print "Welcome " + fullname +"!"
+else:
+	print "The maximum number of login attempts has been reached."
+	print "You could not login anymore."
+
```
Comment (hungphamviet) [inline python_basic/khanhvnv/BI_20_KhanhVu_managing user accounts simple system.py]: This file should not be in scope of this PR. Please remove it.

File: python_basic/khanhvnv/bi_20_managing user accounts simple system_khanhvu.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+		name = raw_input('Login - Enter username: ')
+		passwrd = raw_input('Login - Enter password: ')
+
+	
+if login_success == 1:
+	print "Welcome " + fullname +"!"
+else:
+	print "The maximum number of login attempts has been reached."
+	print "You could not login anymore."
+
```
Comment (hungphamviet) [inline python_basic/khanhvnv/bi_20_managing user accounts simple system_khanhvu.py]: This file should not be in scope of this PR. Please remove it.

File: python_basic/khanhvnv/bi_30_print_out_ an_interesting_square_with_O+and-_khanhvu.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+v_n = input('Please input N: ')
+for i in range(0, v_n):
+    for j in range(0, v_n):
+        if(i==0 or i==v_n-1 or j==0 or j==v_n-1 or i==j):
+            print"O",
+        elif(j>i):
+            print "-",
+        else:
+            print "+",
+    print "\n"
```
Comment (hungphamviet) [inline python_basic/khanhvnv/bi_30_print_out_ an_interesting_square_with_O+and-_khanhvu.py]: Please name this file to be shorter. For example, `print_squares.py` under `python_basic/khanhvnv`.

**Pull request merged**